apps/system/style/chrome/chrome.css

```diff
@@ -386,6 +392,8 @@
 .controls .back-button[disabled],
 .controls .forward-button[disabled] {
   display: none;
+  opacity: 0;
+  animation: fadeOut .2s ease-in;
```

Comment:
Why do we need an animation here? How come we can't just set opacity to 0, and use a transition on the button?